lib/ddtrace/configuration/base.rb

```diff
@@ -5,6 +5,7 @@
 module Datadog
   module Configuration
     # Basic configuration behavior
+    # @public_api
     module Base
```

Comment:
I'm curious about the `@public_api` declarations on this file as well, since customers don't use this directly... (I think...?)